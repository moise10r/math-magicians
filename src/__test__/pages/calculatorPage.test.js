import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorPage from '../../components/pages/calculatorPage';

describe('Calculator page component', () => {
  it('should match the Calculator page', () => {
    const tree = renderer.create(<CalculatorPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
