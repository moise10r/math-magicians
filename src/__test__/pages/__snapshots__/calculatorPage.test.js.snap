// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator page component should match the Calculator page 1`] = `
<section
  className="calculator-main-section"
>
  <div
    className="bg"
  />
  <div
    className="calculator-main-wrapper"
  >
    <div
      className="calculator-container flex-between"
    >
      <div
        className="right-container"
      >
        <p>
          Let do some Math
          <span>
            !
          </span>
        </p>
      </div>
      <div
        className="left-container"
      >
        <div
          className="calculator-main-container"
        >
          <div
            className="row flex-center result"
          >
            <input
              disabled={true}
              onChange={[Function]}
              type="text"
              value="0"
            />
          </div>
          <div>
            <div
              className="row flex-center "
            >
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                AC
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                +/-
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                %
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                รท
              </button>
            </div>
            <div
              className="row flex-center "
            >
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                7
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                8
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                9
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                x
              </button>
            </div>
            <div
              className="row flex-center "
            >
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                4
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                5
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                6
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                -
              </button>
            </div>
            <div
              className="row flex-center "
            >
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                1
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                2
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                3
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                +
              </button>
            </div>
            <div
              className="row flex-center "
            >
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                0
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                .
              </button>
              <button
                className="btn"
                onClick={[Function]}
                type="button"
              >
                =
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
